cmd/go-filecoin/actor_daemon_test.go

```diff
@@ -2,25 +2,29 @@ package commands_test
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/filecoin-project/go-filecoin/cmd/go-filecoin"
-	th "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers"
+	"github.com/filecoin-project/go-filecoin/internal/app/go-filecoin/node/test"
 	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
 )
 
 func TestActorDaemon(t *testing.T) {
 	tf.IntegrationTest(t)
-
+	ctx := context.Background()
 	t.Run("actor ls --enc json returns NDJSON containing all actors in the state tree", func(t *testing.T) {
-		d := th.NewDaemon(t).Start()
-		defer d.ShutdownSuccess()
+		builder := test.NewNodeBuilder(t)
+		n := builder.BuildAndStart(ctx)
+		defer n.Stop(ctx)
+		cmdClient, done := test.RunNodeAPI(ctx, n, t)
+		defer done()
```

Comment:
This pattern is repeated enough that it's probably worth a helper so you can

```
nd, client, done := builder.BuildAndStartAPI(ctx)
defer done()
```
where `done` shuts down both.

Reply:
Great idea.  Because this applies across PRs and is a simple change I'm planning to do this along with the file renames in a PR after all of these land.